raiden_contracts/tests/test_channel_settle.py

```diff
@@ -63,11 +67,121 @@ def test_settle_no_bp_success(
         A,
         0,
         0,
-        locksroot,
+        EMPTY_LOCKSROOT,
         B,
         0,
         0,
-        locksroot,
+        EMPTY_LOCKSROOT,
+    ).transact({'from': A})
+
+
+def test_settle_no_close_bp_success(
```

Comment:
We have a tests for that already [here](https://github.com/raiden-network/raiden-contracts/blob/15d73284a1868307b86b7837cf49c6efb32b364f/raiden_contracts/tests/test_channel_settle.py#L204), and [here](https://github.com/raiden-network/raiden-contracts/blob/15d73284a1868307b86b7837cf49c6efb32b364f/raiden_contracts/tests/test_channel_settle.py#L273)

Reply:
Correct, thank you.
- `test_settle_single_direct_transfer_for_closing_party` (https://github.com/raiden-network/raiden-contracts/blob/15d73284a1868307b86b7837cf49c6efb32b364f/raiden_contracts/tests/test_channel_settle.py#L204) tests what `test_settle_no_update_bp_success` was supposed to test (settlement without `updateNonClosingBalanceProof` being called)

- `test_settle_single_direct_transfer_for_counterparty` (https://github.com/raiden-network/raiden-contracts/blob/15d73284a1868307b86b7837cf49c6efb32b364f/raiden_contracts/tests/test_channel_settle.py#L273) tests what `test_settle_no_close_bp_success test_settle_no_close_bp_success` was supposed to test (settlement without a provided balance proof in `closeChannel`.